RESTResource/RESTResource.js

```diff
@@ -471,8 +477,16 @@ export default class RESTResource {
 
   fetchAction = (props) => {
     const key = this.stateKey();
+
     return (dispatch, getState) => {
-      const options = this.verbOptions('GET', getState(), props);
+      const state = getState();
+      const options = this.verbOptions('GET', state, props);
+
+      if (options.permissionsRequired && !this.hasPerm(state, options.permissionsRequired)) {
+        dispatch(this.actions.fetchAbort({ message: 'cannot satisfy request: missing permissions' }));
```

Comment:
Maybe include the name of the missing permissions in the message?

Reply:
Nice idea @MikeTaylor. I think it will complicate the `hasPerm` a bit but maybe worth it.